sdk/timeseriesinsights/Azure.Iot.TimeSeriesInsights/src/TimeSeriesInsightsClient.cs

```diff
@@ -239,5 +247,777 @@ public virtual Response<TimeSeriesModelSettings> UpdateModelSettingsDefaultTypeI
                 throw;
             }
         }
+
+        /// <summary>
+        /// Gets Time Series instances asynchronously from the environment in pages.
```

Comment:
What do you mean here by "from the environment"?

Reply:
I meant the Time Series environment. Maybe I will remove the "rom the environment" part.